tools/pg_describe.js

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env node
```

Comment:
Any idea why these files don't show up as rename in GitHub? I assume the changes in the files were minor?

Reply:
"minor" in the sense that Prettier was run on them for the first time ever 😛 if it helps make the history cleaner (or at least easier to `git blame`), I can save the rename for another PR.

Reply:
I did this in [c802148](https://github.com/PrairieLearn/PrairieLearn/pull/5799/commits/c8021484ae941fdd9aac538f5f753a49af2f6305).

Reply:
Actually, changed my mind, I'm splitting this up into a few more PRs to keep history clean:
#5800 and #5801.

Reply:
Nice, thanks, I was struggling to see what was going on but now it's crystal clear.

Reply:
The overall diff for this PR should be much clearer now!